front/app/containers/Admin/reporting/containers/ReportBuilder/index.tsx

```diff
@@ -49,6 +49,24 @@ interface Props {
   reportLayout: ReportLayout;
 }
 
+const ROOT_NODE: SerializedNode = {
+  type: 'div',
+  isCanvas: true,
+  props: {
+    id: 'e2e-content-builder-frame',
+  },
+  displayName: 'div',
+  custom: {},
+  hidden: false,
+  nodes: [],
+  linkedNodes: {},
+  parent: undefined as unknown as string, // Bug in CraftJS types
+};
+
+const INITIAL_NODES: CraftJson = {
```

Comment:
Why do we need this object? Is it possible to initialize an empty CraftJs instance without it?

Reply:
Probably, good point. I was trying a lot of different things and ended up with something that worked and forgot about this parrt. But it turns out that it's not necessary, so I've removed it